test/evergreen.yml

```diff
@@ -203,41 +203,49 @@ functions:
     - *configure_wiredtiger
     - command: shell.exec
       params:
-        working_dir: "wiredtiger/build_posix"
+        working_dir: "wiredtiger/cmake_build"
         shell: bash
         script: |
           set -o errexit
           set -o verbose
-          echo '#undef HAVE_FTRUNCATE' >> wiredtiger_config.h
+          echo '#undef HAVE_FTRUNCATE' >> config/wiredtiger_config.h
     - *make_wiredtiger
   "compile wiredtiger address sanitizer":
     - command: shell.exec
       params:
-        working_dir: "wiredtiger/build_posix"
+        working_dir: "wiredtiger"
         shell: bash
         script: |
           set -o errexit
           set -o verbose
-          sh reconf
           if [ "$OS" != "Windows_NT" ]; then
-            CC=/opt/mongodbtoolchain/v3/bin/clang CXX=/opt/mongodbtoolchain/v3/bin/clang++ PATH=/opt/mongodbtoolchain/v3/bin:$PATH CFLAGS="-ggdb -fPIC -fno-omit-frame-pointer -fsanitize=address" CXXFLAGS="-fsanitize=address -fno-omit-frame-pointer -ggdb -fPIC" \
-              ../configure ${configure_python_setting|} \
-              --enable-diagnostic --with-builtins=lz4,snappy,zlib,zstd
+            . test/evergreen/find_cmake.sh
+            if [ -d cmake_build ]; then rm -r cmake_build; fi
+            mkdir -p cmake_build
```

Comment:
The `-p` option might not be necessary since it should not exist or be removed. Happy to keep it as it's harmless.

Reply:
I think it ok to keep around, `-p` is good practice in case these paths change/evolve.